user_docs/en/changes.md

```diff
@@ -2,21 +2,41 @@
 
 ## 2024.4
 
-### Important notes
+This release includes a number of improvements to braille.
+You can now configure how NVDA shows text formatting in braille, and set NVDA to show the start of paragraphs in braille.
+NVDA can now speak the character at the cursor when performing a braille cursor routing action.
+Cursor routing reliability has been improved, and support for routing keys in PowerPoint has been added.
+All lines of cells will now be used when using a multi-line braille display via HID braille.
+The Seika Notetaker driver now correctly generates braille input for space, backspace, and dots with space/backspace gestures.
+NVDA is no longer unstable after restarting NVDA during an automatic Braille Bluetooth scan.
+
+NVDA can now be configured to report font attributes in speech and braille separately.
+
+A new setting has been added to configure the timeout to perform a multiple press gesture, such as the report time/date command.
+
+There have been improvements in Microsoft Word, Excel, and PowerPoint.
+There is now the ability to double-press the comment gesture to read the comment or note in a browsable dialog in Word or PowerPoint.
```

Comment:
I have a suggested alternate wording. More importantly, however: this says PowerPoint. However the actual Change entry below, says Word and Excel.
Is it all three? Or is PowerPoint an error?

```suggestion
In Word or Excel, it is now possible to double-press the comment gesture, to read the comment or note in a browsable dialog.
```

Reply:
You're right, this is only in Word and Excel. Thanks.
```suggestion
In Word or Excel, it is now possible to double-press the comment gesture to read the comment or note in a browsable dialog.
```